lib/configuration.rb

```diff
@@ -109,4 +109,55 @@ def get_optional_class_instance(keyword)
       return Object.const_get(class_opts[:class]).new(**class_opts[:opts])
     end
   end
+
+  class FlexyEnvVariable
+    attr_reader :var_name
+
+    def initialize(var_name)
+      @var_name=var_name
+    end
+    def ==(other)
+      self.class===other &&
+        @var_name==other.var_name
+    end
+    def to_s
+      ENV[@var_name]
+    end
+    
+    alias :eql? :==
+    alias :to_str :to_s
+    alias :[]= :to_s
```

Comment:
do we need this? 

If we need it, shall it be something like (untested):
```
def [](*args, &block)
  self.to_s[*args, &block]
end
```

Reply:
I mean, I can't understand the use case. Maybe what you have is correct, it is just not clear to me what use case it covers.

Reply:
As far as I remember during my work I've got an error like 'can't find a method "[]" for class...', but sure, I'll verify again if the issue is then still here.

Reply:
yes, please send me a backtrace in case you can reproduce

Reply:
Verified, these aliases aren't necessary now thus removing them. It had to be an issue on my side when coding then...